lib/browser/api/exports/electron.js

```diff
@@ -17,6 +17,12 @@ Object.defineProperties(exports, {
       return require('../auto-updater')
     }
   },
+  crashReporter: {
```

Comment:
should probably removed the commented section from common/api/exports/electron.js